SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -186,19 +167,97 @@ QueryConstraint InterestFactory::CreateSystemDefinedConstraints()
 	return SystemDefinedConstraints;
 }
 
-QueryConstraint InterestFactory::CreateUserDefinedConstraints()
+QueryConstraint InterestFactory::CreateCheckoutRadiusConstraints()
```

Comment:
I'm still weary of using NetCullDistance in this way. The default value Unreal has is MASSIVE, so I'm not sure if we want that.

Reply:
For observers, we discussed this and agree. In addition, since the value is a square of a cm distance, it doesn't take much to start hitting floating point precision errors when sqrt-ing. Refactoring to a a non-squared property on AActors.